plugins/pluginmanagers/lz-n.nix

```
@@ -0,0 +1,126 @@
+{
+  lib,
+  helpers,
+  config,
+  pkgs,
+  ...
+}:
+with lib;
+let
+  lzPluginType =
+    with helpers.nixvimTypes;
+    let
+      maybeRawStrOrList = either (maybeRaw str) (listOf (maybeRaw str));
```

```suggestion
      maybeRawStrOrList =
        oneOf [
          str
          (listOf (maybeRaw str))
          rawLua
        ];
```

This is the same thing, but with fewer nested `either`s (`maybeRaw` is just `t: either t rawLua`)